manifest.toml

```diff
@@ -0,0 +1,73 @@
+#:schema https://raw.githubusercontent.com/YunoHost/apps/master/schemas/manifest.v2.schema.json
+
+packaging_format = 2
+
+id = "hotspot"
+name = "Wifi Hotspot"
+description.en = "Create and configure a WiFi hotspot"
+description.fr = "Créez et gérez un point d'accès WiFi"
+
+version = "2.3.0~ynh1"
+
+maintainers = []
+
+[upstream]
+license = "AGPL-3.0"
+website = "https://internetcu.be/"
+
+[integration]
+yunohost = ">= 11.2"
+architectures = "all"
+multi_instance = true
+ldap = "not_relevant"
+sso = "not_relevant"
+disk = "50M"
+ram.build = "50M"
+ram.runtime = "50M"
+
+[install]
+    [install.wifi_ssid]
+    ask.en = "Choose a wifi name (SSID)"
+    ask.fr = "Choisissez un nom pour le wifi (SSID)"
+    type = "string"
+    example = "myNeutralNetwork"
+    default = "myNeutralNetwork"
+
+    [install.wifi_passphrase]
+    ask.en = "Choose a wifi password (at least 8 characters for WPA2)"
+    ask.fr = "Choisissez un mot de passe wifi (au minimum 8 caractères pour le WPA2)"
+    type = "password"
+
+    [install.firmware_nonfree]
+    ask.en = "Install non-free WiFi firmwares? (Only needed if you're using a proprietary WiFi antenna/dongle)"
+    ask.fr = "Installer des firmwares WiFi non-libres ? (Nécessaire seulement si vous utilisez une antenne/clé WiFi propriétaire)"
+    type = "boolean"
+    default = false
+
+[resources]
+    [resources.system_user]
+
+    [resources.apt]
+    packages = "sipcalc, hostapd, iw, kmod"
+
+# =========================================
+# FIXME FIXME FIXME FIXME FIXME FIXME FIXME 
+#   Gotta find a way to conditionally install the non-free packages
+#  So far the "packages_from_raw_bash" thing doesnt allow conditional packages from extra repo (here, the non-free component..)
+# FIXME FIXME FIXME FIXME FIXME FIXME FIXME 
```

Comment:
Gotta find something for this ... it probably implies to add the possibility inside the core x_x